index.js

```diff
@@ -1 +1,120 @@
 // implement your API here
+const express = require('express');
+const db = require('./data/db');
+const server = express();
+server.use(express.json());
+
+// Delivers 'server running' response:
+server.get('/', (req, res) => {
+  res.send('Server running...');
+});
+
+// Fetches users array:
+server.get('/api/users', (req, res) => {
+  db.find()
+    .then(users => {
+      if (users) {
+        res.status(200).json(users);
+      } else {
+        res.status(404).json({ message: 'no users to return' });
+      }
+    })
+    .catch(err => {
+      res
+        .status(500)
+        .json({ error: 'The users information could not be retrieved.' });
+    });
+});
+
+// Fetches a user:
+server.get('/api/users/:id', (req, res) => {
+  const id = req.params.id;
```

Comment:
don't forget you can also destructure here to save yourself, like, one net keystroke :laughing: